docs/DesignDocumentation.md

```diff
@@ -7,6 +7,20 @@ nav_order: 4
 
 While designing VulnerableApp, major emphasis was given on **Ease of adding Vulnerabilities** such that developers of Vulnerability Scanners need to put minimal effort for adding new Vulnerabilities for testing their payload/attack vectors.
 
+## Technologies used
+- Java8
+- Spring Boot
+- Vanilla Javascript
+- CSS
+- Inmemory H2 database
+- Other small libraries used are:
+    - Unit Testing libraries:
+        - mockito
+        - AssertJ
+        - jupiter
+    - nimbus-jose library for jwt
+#### Note: we are not limited to these technologies and if required open to expand to other technologies.
+
```

Comment:
I feel it would be sufficient to list the app as
Spring boot - JavaScript
Other implementation level details might be implicitly understood ?

Reply:
hmm sure.